src/app/services/data.service.ts

```diff
@@ -1,14 +1,25 @@
 import { Observable, Subscriber } from 'rxjs';
 import * as io from 'socket.io-client';
+import { environment } from 'src/environments/environment';
 
 export class DataService {
-  private url = 'http://localhost:3000/sensor';
+  private url = `${environment.apiUrl}`;
   private socket: SocketIOClient.Socket;
 
   constructor() { }
 
   public connect() {
-    this.socket = io(this.url);
+    // socket.io-client excepts the url to be in the form of host/namespace. This doesn't work when the backend has the
```

Comment:
except > expects?

Reply:
excepts is Python 🐍 Dit moest inderdaad expects zijn 😂